framework/web_app/src/services/api/ProductApiService.ts

```diff
@@ -12,31 +13,20 @@ export default class ProductApiService {
         this.mapiHttpClient = new AxiosHttpClient(5172);
     }
 
-    createAsync = async (command: CreateProductCommand): Promise<CreatedResponse> =>
-        await this.dapiHttpClient.post<CreatedResponse>('/products', command);
+    createAsync = async (product: Partial<Product>): Promise<CreatedResponse> =>
```

Comment:
i think we chatted about this a bit

issue i have with it is it only matches the use case requirements for now, but in the future we may have a mish mash of request requirements where request A uses the model because it fits the API's endpoint definition, and request B uses a command object

may get confusing? always found explicitly using separate types easier to read

it does mean we increase the amount of objects to maintain though

Reply:
had this debate in my head when i first created this command object here, and came to the conclusion that no the domain entity will not always match up with the API command

Reply:
maybe do class

Reply:
added back command types